Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/docrepository/adapter/model/DocumentMetadata.java

```diff
@@ -218,6 +218,9 @@ public String getDocumentUniqueId() {
     }
 
     public void setDocumentUniqueId(String documentUniqueId) {
+        if (null != document) {
+            document.setDocumentUniqueId(documentUniqueId);
```

Comment:
Do we need a not null check for documentUniqueId

Reply:
In this instance for the test data loader it's a required field in the admin gui. So it can't be null.